modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/java/com/liferay/content/dashboard/document/library/internal/item/FileEntryContentDashboardItem.java

```diff
@@ -215,6 +223,44 @@ public ContentDashboardItemSubtype getContentDashboardItemSubtype() {
 		return _contentDashboardItemSubtype;
 	}
 
+	@Override
+	public List<ContentDashboardItemVersionAction>
```

Comment:
Hey I think the getContentDashboardItemVersionActions should be added to the ContentDashboardItemVersion, instead the ContentDashboardItem implementation.